web/packages/hovercards/src/core.ts

```diff
@@ -419,6 +423,7 @@ export default class Hovercards {
 								label: account.service_label,
 								icon: account.service_icon,
 								url: account.url,
+								isHidden: account.is_hidden,
```

Comment:
This will always be present on `account`?

Reply:
Has replied on the relevant Slack thread.